app/uk/gov/hmrc/eusubsidycompliancestub/models/Subsidy.scala

```diff
@@ -25,9 +25,9 @@ case class Subsidy(
   subsidyUsageTransactionId: Option[SubsidyRef],
   allocationDate: LocalDate,
   submissionDate: LocalDate,
-  publicAuthority: String, // no regex
-  traderReference: Option[TraderRef], // no regex in create API but one in retrieve API!
-  nonHMRCSubsidyAmtEUR: PositiveSubsidyAmount, // TODO consider using sane names and write a bespoke formatter
```

Comment:
It's a stub for an external API on which we don't have control, it would break the functionality to rename this.